tutor4points/core/templates/home.html

```diff
@@ -18,11 +20,39 @@
             <div class="dashboard-container d-flex flex-row">
                 <h2>Account Balance: {{ user.total_points }} points</h3>
             </div>
-            <div class="dashboard-container">
-                <div class="list-container">
-                    <h2>Tutor Requests</h2>
+            {% if user.is_tutor %}
+                <div class="dashboard-container">
```

Comment:
I just created the tutor requests page and have the accept/decline buttons incorporated. You could take that and it to the dashboard either now or in the future.